test/unit/lib/plugins/aws/provider.test.js

```diff
@@ -274,80 +274,6 @@ describe('AwsProvider', () => {
     });
   });
 
-  describe('#getDeploymentPrefix()', () => {
-    it('should return custom deployment prefix if defined', () => {
-      serverless.service.provider.deploymentPrefix = 'providerPrefix';
-
-      expect(awsProvider.getDeploymentPrefix()).to.equal(
-        serverless.service.provider.deploymentPrefix
-      );
-    });
-
-    it('should use the default serverless if not defined', () => {
-      serverless.service.provider.deploymentPrefix = undefined;
-
-      expect(awsProvider.getDeploymentPrefix()).to.equal('serverless');
-    });
-
-    it('should support no prefix', () => {
-      serverless.service.provider.deploymentPrefix = '';
-
-      expect(awsProvider.getDeploymentPrefix()).to.equal('');
-    });
-  });
-
-  describe('#getAlbTargetGroupPrefix()', () => {
-    it('should return custom alb target group prefix if defined', () => {
-      serverless.service.provider.alb = {};
-      serverless.service.provider.alb.targetGroupPrefix = 'myPrefix';
-
-      expect(awsProvider.getAlbTargetGroupPrefix()).to.equal(
-        serverless.service.provider.alb.targetGroupPrefix
-      );
-    });
-
-    it('should return empty string if alb is not defined', () => {
-      serverless.service.provider.alb = undefined;
-
-      expect(awsProvider.getAlbTargetGroupPrefix()).to.equal('');
-    });
-
-    it('should return empty string if not defined', () => {
-      serverless.service.provider.alb = {};
-      serverless.service.provider.alb.targetGroupPrefix = undefined;
-
-      expect(awsProvider.getAlbTargetGroupPrefix()).to.equal('');
-    });
-
-    it('should support no prefix', () => {
-      serverless.service.provider.alb = {};
-      serverless.service.provider.alb.targetGroupPrefix = '';
-
-      expect(awsProvider.getAlbTargetGroupPrefix()).to.equal('');
-    });
-  });
-
-  describe('#getAccountInfo()', () => {
```

Comment:
Is there a replacement test for this removed one? Or it was removed by mistake?

Reply:
This is a mistake :disappointed: Thanks for catching it :bow: